LSE-61.tex

```diff
@@ -1174,6 +1178,93 @@ \subsubsection{All-Sky Visualization of Data Releases}
+
+<br>By the terms of the HiPS Recommendation, a HiPS image map should include a corresponding MOC.  This may or may not be the same as the MOCs for the survey envisioned under \hyperref[DMS-REQ-0383]{DMS-REQ-0383} elsewhere in this document, depending on choices made for data selection.
+
+<br>The Project should produce a technical note, during the construction era, detailing which of the optional components of the HiPS standard will be supported.
+
+<br>This requirement specifically calls for making HiPS maps from the standard coadds and therefore whatever policies are used for the inclusion of Special Programs data in the standard coadds will also automatically apply here.  If there are both main-survey-depth and full-depth coadds for the deep drilling fields, then, it is a separate question as to whether HiPS maps will be generated for those fields.
+
+\paragraph{HiPS Service}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0380}
+\textbf{ID:} DMS-REQ-0380
+
+\textbf{Specification:}
+The Data Management system shall include a secure and authenticated Internet endpoint for an IVOA-compliant HiPS service.  This service shall be advertised via Registry as well as in the HiPS community mechanism operated by CDS, or whatever equivalent mechanism may exist in the LSST operations era.
+
+\textbf{Discussion:}
+The DM HiPS service will be available only to data rights holders.  LSST EPO will also operate a world-public HiPS service, but with its spatial resolution limited to approximately one arcsecond.
+
+\paragraph{HiPS Linkage to Coadds}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0381}
+\textbf{ID:} DMS-REQ-0381
+
+\textbf{Specification:}
+The HiPS maps produced by the Data Management system shall provide for straightforward linkage from the HiPS data to the underlying LSST coadded images.  This SHOULD be implemented using a mechanism supported by both the LSST Science Platform and by community tools.
+
+\textbf{Discussion:}
+It is intended that this be done using the "HiPS Progenitor" mechanism introduced at the May 2018 IVOA meeting, or an evolution of it that emerges from the IVOA standardization process.
+
+\paragraph{HiPS Visualization}\hfill  % Force subsequent text onto new line
+
+\label{DMS-REQ-0382}
+\textbf{ID:} DMS-REQ-0382
```

Comment:
Priority 1b and OSS-REQ-0061.  OSS-REQ-0061 has two specifically relevant subclauses.